src/main/resources/application.yml

```diff
@@ -28,6 +28,7 @@ janitor:
     force-clean-start: false
     max-parallel-flights: 40
     migrate-upgrade: true
+    name: janitor-stairway
```

Comment:
ditto other PR comment about moving this to the default value and explaining its importance.